cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -587,11 +604,12 @@ def __init__(self, theta: cirq.TParamVal, phi: cirq.TParamVal) -> None:
         )
 
     @property
-    def phi(self) -> cirq.TParamVal:  # pylint: disable=missing-function-docstring
+        """Angle (in radians) defining the axis of rotation in the `X`-`Y` plane."""
         return self.phase_exponent * _pi(self.phase_exponent)
```

Comment:
Deleted function header:
```suggestion
    def phi(self) -> cirq.TParamVal:
        """Angle (in radians) defining the axis of rotation in the `X`-`Y` plane."""
        return self.phase_exponent * _pi(self.phase_exponent)
```